fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java

```diff
@@ -298,7 +299,11 @@ public Void computeHMSTableScanNode(Operator node, ExpressionContext context, Ta
         List<ColumnRefOperator> requiredColumns = new ArrayList<>(colRefToColumnMetaMap.keySet());
 
         List<ColumnStatistic> columnStatisticList;
+
         try {
+            if (!optimizerContext.getSessionVariable().enableHiveColumnStats()) {
+                throw new Exception("Session variable " + SessionVariable.ENABLE_HIVE_COLUMN_STATS + " is false");
```

Comment:
Why throw an exception here？

Reply:
To goto following catch scope: use unknown column stats. Yes it's quite unusual, better to use if-condition stmt.

Reply:
done